docs/installation.rst

```diff
@@ -18,6 +18,9 @@ Please get in touch
 by `opening an issue <https://github.com/adc-connect/adcc/issues>`_
 if you cannot get adcc to work.
 
+Some specialty features of adcc require extra dependencies (Pandas and
```

Comment:
I would add an `e.g.`, that ensure the docs don't become outdated so easily (one quickly forgets to add a dependency here, while it's clear to put it to the docs below.

Reply:
Done.